src/sdk/trace/config.rs

```diff
@@ -17,6 +17,8 @@ pub struct Config {
     pub max_attributes_per_span: u32,
     /// The max links that can be added to a `Span`.
     pub max_links_per_span: u32,
+    /// Contains attributes representing an entity that produces telemetry.
+    pub resource: sdk::Resource,
```

Comment:
Might actually be better for this to be `Arc<sdk::Resource>`

Reply:
Could save a lot of cloning, especially if there are many attributes associated with the resource. But it would impact the `Serialize`/`Deserialize` impl as you can't deserialize an `Arc<T>`.

@MikeGoldsmith / @jodh-intel thoughts on this?

Reply:
Nvm, serde's `rc` feature actually works fine in this case.